CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+##2.0.0
+
+###Changed
+* Using version 2 of the notification-api.
```

Comment:
need to include no longer being able to use separate `service_id` and `api_key_id` in this